debian/rules

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/make -f
+
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+export PYBUILD_NAME=jetson-gpio
+export PYBUILD_DISABLE=test
+
```

Comment:
Since we are packaging the udev rules in the common package, we don't need the file in the Python packages. So, let's add this here:
export PYBUILD_AFTER_INSTALL=find '{destdir}/{install_dir}/' -name 99-gpio.rules | xargs rm

(There are probably other ways to do this, such as hacking setup.py to detect when a Debian package is being built and telling it not to install 99-gpio.rules in the first place, but this should work fine for now.)